internal/plugin/worker.go

```diff
@@ -995,6 +996,48 @@ func (m *Manifest) genWorkerWorkflowInput(f *j.File, workflow protoreflect.FullN
 			g.Id(m.methods[signal].GoName).Op("*").Add(m.Qual(signal, m.toCamel("%sSignal", signal)))
 		}
 	})
+
+	f.Comment("ContinueAsNew returns an appropriately configured ContinueAsNewError")
+	f.Func().
+		ParamsFunc(func(g *j.Group) {
+			g.Id("i").Op("*").Id(typeName)
+		}).
+		Id("ContinueAsNew").
+		ParamsFunc(func(g *j.Group) {
+			g.Id("ctx").Qual(workflowPkg, "Context")
+			if hasInput {
+				g.Id("nextInput").Op("...").Op("*").Qual(string(method.Input.GoIdent.GoImportPath), m.getMessageName(method.Input))
```

Comment:
do you currently have convention for optional argument via spread vs nil? I think Temporal's official SDK tend to embrace "pass nil if you want to use defaults" so wondering if the spread operator here is intentional